docs/modules/ROOT/pages/cli/cli-sbom.adoc

```diff
@@ -0,0 +1,133 @@
+= Download SBOMs
+
+:toc: left
+:icons: font
+:numbered:
+:source-highlighter: highlightjs
+
+== Software Bill of Materials
+A software bill of materials (SBOM) provides greater transparency for your software supply chain. In {ProductName}, an SBOM lists all the software libraries that a xref:glossary/index.adoc#_component[component] uses. Those libraries may enable specific functionality, facilitate development, or serve some other purpose. 
+
+You can use an SBOM to better understand your software’s composition, identify vulnerabilities, and assess the potential impact of any security issues that may arise. You may also need to provide a component’s SBOM to comply with cybersecurity regulations. 
+
+== Downloading an SBOM
+
+*Prerequisites*
+
+* Install the link:https://docs.sigstore.dev/cosign/installation/[Cosign] CLI tool.
+
+* Install the link:https://stedolan.github.io/jq/download/[jq] CLI tool.
+
+* xref:cli/getting_started_in_cli.adoc[Login] to {ProductName}.
+
+*Procedure*
+
+Complete the following steps to download a component's SBOM:
+
+. List your components.
++
+[source]
+----
+$ oc get components
+----
++
+Example output:
+
++
+[source]
+----
+NAME                               AGE   STATUS   REASON   TYPE
+devfile-sample-go-basic-8wqt       8m54s True     OK       Updated
+devfile-sample-python-basic-ikch   20d   True     OK       Updated
+----
+
+. Choose which component's SBOM you want to download. Then use `oc get` and jq to get that component's image path.
+
++
+[source]
+----
+$ oc get component <component name> -ojson | jq '.status.containerImage'
+----
+
++
+Example:
+
++
+[source]
+----
+$ oc get component devfile-sample-python-basic-ikch -ojson | jq '.status.containerImage'
+    "quay.io/redhat-appstudio/user-workload@sha256:<output omitted>"
+----
+
+. Use Cosign to download the SBOM. Pass the image path, from the output of the last command, as an argument into Cosign's `download sbom` command. Be sure to delete any quotation marks around the image path.
+
++
+[source]
+----
+$ cosign download sbom <image path>
+----
+
++
+Example:
+
++
+[source]
+----
+$ cosign download sbom quay.io/redhat-appstudio/user-workload@sha256:<output omitted>
+----
```

Comment:
would it be worth to add a note that cosign might print out this warning after running the command and state if user should (not) worry about it?
```
❯ cosign download sbom quay.io/psturc/test-images@sha256:51f321b27c2c0b3669b03fea4c889575cd72b66f4a741b7cdeaf21439f8fb5df > /tmp/sbom.txt
WARNING: Downloading SBOMs this way does not ensure its authenticity. If you want to ensure a tamper-proof SBOM, download it using 'cosign download attestation <image uri>' or verify its signature using 'cosign verify --key <key path> --attachment sbom <image uri>'.
Found SBOM of media type: application/vnd.cyclonedx+json

❯ cosign version
...
GitVersion:    v2.0.0
```

Reply:
Great suggestion--I added a note about that.